pkg/api/pluginproxy/ds_proxy.go

```diff
@@ -346,13 +347,18 @@ func (proxy *DataSourceProxy) logRequest() {
 
 	panelPluginId := proxy.ctx.Req.Header.Get("X-Panel-Plugin-Id")
 
+	uri, err := loggermw.SanitizeURL(proxy.ctx.Req.RequestURI)
+	if err == nil {
+		proxy.ctx.Logger.Warn("Could not sanitize RequestURI in ds proxy due to: %e", err)
```

Comment:
Should I log this in some different way?

Reply:
```suggestion
		proxy.ctx.Logger.Error("Could not sanitize RequestURI", "error", err)
```

Maybe also an idea here to set `uri` to something like `"<redactedDueToSanitizationError>"` or something? 

Based on your comment:
>Should we, instead of just removing the value, replace the value with some kind of masking (*****) to make it obvious to someone going through the logs that the information has been masked due to security reasons?

I think that could be a nice alternative 👍